### PR TITLE
Fix broken method card links in discovery section

### PR DESCRIPTION
**Pull request by jseppi**

### CODE REVIEW

File: pages/1-discovery-research.md [hunk 1]
```diff
@@ -6,12 +6,13 @@ The first stage of any project is to do research to discover problems that need
...
->- [Contextual inquiry](https://methods.18f.gov/contextual-inquiry/)[](https://methods.18f.gov/bodystorming/) 
-- [Stakeholder and user interviews](https://methods.18f.gov/stakeholder-and-user-interviews/) 
-- [Journey mapping](https://methods.18f.gov/journey-mapping/) 
-- [Cognitive walkthroughs](https://methods.18f.gov/cognitive-walkthrough/) 
-- [KJ method](https://methods.18f.gov/kj-method/) 
-- [Heuristic analysis ](https://methods.18f.gov/heuristic-analysis/) 
+- [Contextual inquiry](https://methods.18f.gov/discover/contextual-inquiry/)
+- [Bodystorming](https://methods.18f.gov/discover/bodystorming/) 
+- [Stakeholder and user interviews](https://methods.18f.gov/discover/stakeholder-and-user-interviews/) 
+- [Journey mapping](https://methods.18f.gov/decide/journey-mapping/)
```
Comment (jseppi) [inline pages/1-discovery-research.md:12]: According to methods.18f.gov, "journey mapping" is part of the "Decide" group of methods, so perhaps it should not be included in this list?

File: pages/1-discovery-research.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ The first stage of any project is to do research to discover problems that need
 
 These design methods can help guide your research with both stakeholders and possible end users:
 
->- [Contextual inquiry](https://methods.18f.gov/contextual-inquiry/)[](https://methods.18f.gov/bodystorming/)
```
Comment (jseppi) [inline pages/1-discovery-research.md:9]: Not sure why the `>` was before this list item. I noticed it in other lists as well.

**Comment by typesend** [comment]
Come on, @brethauer! This totally was tripping me up! LGTM!

image

**Comment by brethauer** [comment]
Thanks for the fix @jseppi and thanks for the nudge @typesend ;)

**Pull request merged**